third_party/plaidml/workspace.bzl

```diff
@@ -0,0 +1,20 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def clean_dep(dep):
+    return str(Label(dep))
+
+def repo():
+    http_archive(
+        name = "com_intel_plaidml",
+        build_file = clean_dep("//third_party/plaidml:BUILD.bazel"),
+        sha256 = "d97ac6d59f3a335c91d224a6cccd326c1aa057d779d89ba17f7cc4e00769c39a",
+#        sha256 = "899df7cccfcd26dc3a33dfd7eba97e34ba115fd65910d7631e0feede0c277e53",
+        url = "https://github.com/plaidml/plaidml/archive/plaidml-v1.zip",
```

Comment:
Use a specific commit, this uses whatever is on the HEAD of the `plaidml-v1` branch (which is constantly moving).